scripts/maintenance/computational-clusters/Makefile

```diff
@@ -0,0 +1,15 @@
+.DEFAULT_GOAL := install
+
+SHELL := /bin/bash
```

Comment:
Out of curiosity: Why do you set this variable? It looks to me like it is not used below

Reply:
SHELL? this defines what shell the GNU Make shall use. by default it uses posix shell or ```sh```

Reply:
aha, good to know. I still have a lot of Make to learn. Thanks!